node-src/tasks/index.ts

```diff
@@ -30,3 +31,20 @@ export default (options: Context['options']): Listr.ListrTask<Context>[] => {
 
   return options.junitReport ? tasks.concat(report) : tasks;
 };
+
+export type GitInfo = {
+  branch: string;
+  commit: string;
+  slug: string;
+};
+
+export async function getGitInfo(): Promise<GitInfo> {
+  const branch = await getBranch();
+  const { commit } = await getCommit();
+  const slug = await getSlug();
+
+  const [ownerName, repoName, ...rest] = slug ? slug.split('/') : [];
+  const isValidSlug = !!ownerName && !!repoName && !rest.length;
+
+  return { branch, commit, slug: isValidSlug ? slug : '' };
+}
```

Comment:
This isn't a task, I think it should be in `node-src/index.ts` (in fact I'm unclear on how this actually works exported from here).

Reply:
Ow, I put it in the wrong file 😓